PlayBook/src/getting_started/DownloadIPA.md

```diff
@@ -0,0 +1,42 @@
+# Download & Install .ipas
+
+.ipas must be decrypted to install them with PlayCover. Quite often, the app you're looking for will already be decrypted.
```

Comment:
".ipas" might make some people believe that is the actual extension. It is weird to start a sentence with a dot, so maybe it could be changed to "IOS .ipa files" or the like.